src/aioswitcher/api/__init__.py

```diff
@@ -57,15 +57,17 @@
 
 logger = getLogger(__name__)
 
-SWITCHER_TCP_PORT2 = 10000
-SWITCHER_TCP_PORT = 9957
+# Type 1 devices: Heaters (v2, touch, v4), Plug
+SWITCHER_TCP_PORT_TYPE1 = 9957
```

Comment:
If these are static ports, why not include them in the var name?
```python
SWITHCER_TCP_PORT_9957 = 9957
SWITCHER_TCP_PORT_10000 = 10000
```
?

Reply:
Yeah but the port itself is just an integer and is not directly related to the type of device using it as its broadcast port.
For future contributors, or even for our future self, "SWITCHER_TCP_PORT2" doesn't have too much meaning to it if you're not aware and/or make the connection that this correlated with the TYPE2 device.